File: C++/Daily Flash/Week2/Day3_Jan22/code/prog5.cpp
#include <iostream>

int main() {
	int row;
	std::cout << "Enter rows: " << std::endl;
	std::cin >> row;

	for(int olc=0; olc<row; olc++){
		for(int ilc=0; ilc<=olc; ilc++)
			std::cout << " * ";
		std::cout << std::endl;
	}
	return 0;

}

File: JAVA/Daily Flash Codes/Week1/Day6/code_Java/prog5.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		System.out.println("Enter row: ");
		int row = Integer.parseInt(in.readLine());

		for(int olc = 0 ;olc < row; olc++) {
			for(int ilc = 0; ilc < row ; ilc++)
				System.out.print(" 0 ");
			System.out.println();
		}


	}


}
